### ISSUE TITLE
unicode string can't be autoescaped by render

### ISSUE THREAD

**Issue by leafsummer** [elapouya/python-docx-template#159]
the code is
```python
if isinstance(v, str):
     # Unescape at first to avoid secondary escaping
     d[k] = escape(unescape(v))
```

when the v is the unicode, for example：u"\u4f60&\u6211", the escape can not handle it. is it added unicode type to if judgment.
```python
if isinstance(v, (str, unicode)):
     # Unescape at first to avoid secondary escaping
     d[k] = escape(unescape(v))
```

**Comment by mx2048** [comment]
Are you using Python 2.7? 
The escape method takes care of XML-reserved characters only. These are <, >, ", ', and &.

**Comment by leafsummer** [comment]
> Are you using Python 2.7?
> The escape method takes care of XML-reserved characters only. These are <, >, ", ', and &.

yes, I'm using Python 2.7. my document context is a unicode string. when to use the autoescape in render, I found the escape function can not work.

**Comment by mx2048** [comment]
I'll try to fix that.

**Comment by mx2048** [comment]
I pull-requested to fix issue #159.
See `tests/escape_auto.py`, I added your Unicode example.
Enjoy!

**Comment by elapouya** [comment]
fixed in 0.5.14
